src/utils.py

```diff
@@ -157,4 +163,39 @@ def convert_dtypes(df):
     placeholders = ', '.join(len(df.columns)*['%s'])
     logger.info('Datatypes converted from python to mysql.')
     
-    return types, placeholders
\ No newline at end of file
+    return types, placeholders
+
+
+def upload_to_s3(df: pd.DataFrame, bucket_name: str, file_name: str):
+    """_Method to upload files to s3 bucket.
+
+    Parameters
+    ----------
+    s3 : Connection to bucket.
+    file : Filepath for file to be uploaded to bucket.
+    bucket_name : Bucket name where file to be uploaded.
+    object_name : Key to the file to identify in bucket.
+    """
+    s3 = aws_auth()
+    csv_data = df.to_csv(index = False)
+    s3.put_object(Body = csv_data,
+                  Bucket = bucket_name,
+                  Key = f'{file_name}.csv')
+    logger.info(f'Successfully Uploaded data in {bucket_name} as {file_name}')
+    
+
+def download_from_s3(bucket,key_name,file_name):
```

Comment:
this function is incomplete.
When you do get_object, it returns a json object. You need to store that json object into a variable and extract data from it.

```
response = s3.get_object(bucket,key_name,file_name)
df = pd.read_csv(response['Body'])
return df
```